src/Symfony/Component/JsonEncoder/PhpPrinter.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\JsonEncoder;
+
+use PhpParser\Node\Expr\Closure;
+use PhpParser\Node\Expr\Yield_;
+use PhpParser\PhpVersion;
+use PhpParser\PrettyPrinter\Standard;
+
+if (!class_exists(PhpVersion::class)) {
```

Comment:
What's the pro in supporting v4? Could it require php-parser:^5 instead?

Reply:
When that PR was opened, the v5 version of php-parser was brand-new, that's why the compatibility with the v4 was kept.
But now, the v5 its widely used, so I dropped the v4 support, thanks!